BrowserKit/Sources/WebEngine/WKWebview/WKEngineSession.swift

```diff
@@ -116,6 +125,15 @@ class WKEngineSession: NSObject,
         webView.engineScrollView.setContentOffset(CGPoint.zero, animated: true)
     }
 
+    func findInPage(text: String, function: FindInPageFunction) {
+        let sanitizedInput = text.replacingOccurrences(of: "\\", with: "\\\\").replacingOccurrences(of: "\"", with: "\\\"")
```

Comment:
Just OOC is this something being brought over from the existing client? Wondering if there is a more elegant way for us to avoid how we're escaping these slashes but I'm not super-familiar with this code.

Reply:
Yeah it's brought from the existing code 🤔 It's from [here](https://github.com/mozilla-mobile/firefox-ios/blob/31ebf712886a669a298f040a4dbc7b0877dbfa27/firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController%2BFindInPage.swift#L77-L78) exactly